appengine-managed-runtime/src/main/java/com/google/apphosting/vmruntime/VmApiProxyDelegate.java

```diff
@@ -159,6 +166,37 @@ public VmApiProxyDelegate() {
           "complete. Service bridge status code: %s; response " +
           "content-length: %s. Took %s ms.", packageName, methodName, requestData.length, 200,
           responseData.length, (end - start)));
+      
+      // HACK TO FIX USER_SERVICE ISSUE #164
+      if ("user".equals(packageName)) {
+        boolean userservicefix = !Boolean.getBoolean("UserService164");
```

Comment:
yes.... Maybe useLocalUserService ?
And true by default

On Tue, Feb 23, 2016 at 12:46 PM, Greg Wilkins <notifications@github.com>
wrote:

> In
> appengine-managed-runtime/src/main/java/com/google/apphosting/vmruntime/VmApiProxyDelegate.java
> <https://github.com/GoogleCloudPlatform/appengine-java-vm-runtime/pull/182#discussion_r53847044>
> :
>
> > @@ -159,6 +166,37 @@ public VmApiProxyDelegate() {
> >            "complete. Service bridge status code: %s; response " +
> >            "content-length: %s. Took %s ms.", packageName, methodName, requestData.length, 200,
> >            responseData.length, (end - start)));
> > +
> > +      // HACK TO FIX USER_SERVICE ISSUE #164
> > +      if ("user".equals(packageName)) {
> > +        boolean userservicefix = !Boolean.getBoolean("UserService164");
>
> System prop to disable has been added. @ludoch <https://github.com/ludoch>
> maybe a bad name?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/appengine-java-vm-runtime/pull/182/files#r53847044>
> .
>